src/app/files/browser-files.js

```diff
@@ -11,15 +11,13 @@ function Files (storage) {
   this.exists = function (path) {
     var unprefixedpath = this.removePrefix(path)
     // NOTE: ignore the config file
-    if (path === '.remix.config') {
-      return false
-    }
+    if (path === '.remix.config') return false
 
     return this.isReadOnly(unprefixedpath) || storage.exists(unprefixedpath)
   }
 
   this.init = function (cb) {
-    cb()
+    this.resolveDirectory('', (error, filesTree) => cb && cb(error))
```

Comment:
not sure we need to `resolveDirectory` in the init function

Reply:
yes, probably it's not necessary. I will try without - i don't know why i thought it was necessary - i have to get back into it :-)